packages/e2e/cypress/integration/ui/components/geo/map-with-geocoder/map-with-geocoder.steps.ts

```diff
@@ -13,7 +13,7 @@ When('I press the enter key', () => {
 });
 
 When('I click on a map marker', () => {
-  cy.get('.maplibregl-marker').first().click();
+  cy.get('.maplibregl-marker').first().click({ force: true });
```

Comment:
The error i saw in my [test PR](https://github.com/aws-amplify/amplify-ui/pull/1806) was:
```
  1) Map with Geocoder
       Marker information popup:
     CypressError: Timed out retrying after 15050ms: `cy.click()` failed because this element:

`<div aria-label="Map marker" class="maplibregl-marker mapboxgl-marker maplibregl-marker-anchor-center mapboxgl-marker-anchor-center" tabindex="0" style="transform: translate(-50%, -50%) translate(674px, 272px) rotateX(0deg) rotateZ(0deg);">...</div>`

is being covered by another element:

`<path d="M27,13.5 C27,19.074644 20.250001,27.000002 14.75,34.500002 C14.016665,35.500004 12.983335,35.500004 12.25,34.500002 C6.7499993,27.000002 0,19.222562 0,13.5 C0,6.0441559 6.0441559,0 13.5,0 C20.955844,0 27,6.0441559 27,13.5 Z"></path>`

Fix this problem, or use {force: true} to disable error checking.
```

Somewhat concerned with force clicking since it implies that there is a covering element, do you think that this could lead to false positives in the future?

Reply:
Since I can't reproduce it outside of the e2e tests, by manually clicking, this doesn't seem to be a real issue. But we could check with @slaymance if he has any guidance.

Reply:
It seems that the documentation on the cypress site states that force, forces the click, instead of waiting for `actionabality`

<img width="729" alt="image" src="https://user-images.githubusercontent.com/65630/166294227-6b438c5d-4d06-44ed-93a5-6ab34df70fd2.png">


So it is abled to be clicked, it just gets confused because of the SVG icon. I don't think this would cause any `false` positives in the future, and we only use this method in one place so far. 

I can't predict how this `When` will be used in the future.